docker-compose-config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Add `set -ex` 

https://www.gnu.org/software/bash/manual/bash.html#Modifying-Shell-Behavior

Note you might need to refactor the below

Reply:
Hm, not sure I understand, why so?

Reply:
-e means that the pipeline will exit if commands fail - if we fail to install pyyaml because e.g. python 3 is not installed this script should not go on.

-x means we echo the commands - if stuff fails, this makes debugging easier.

Both are pretty standard to set in all bash scripts.